hello_world.py

```diff
@@ -0,0 +1,2 @@
+print("Hello, World!")
```

Comment:
@TanyaaCJain , please add another exclaimation mark

Reply:
@delta619 Fixed it.